introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js

```diff
@@ -0,0 +1,130 @@
+const $body = document.querySelector('body');
+const $buttonAddInput = document.querySelector('#add-input-button');
+const $buttonRemoveInput = document.querySelector('#remove-input-button');
+const $salaryFormContent = document.querySelector('#salary-form-content');
+const $salaryForm = document.querySelector('#salary-form');
+const $buttonCalculateSalary = document.querySelector('#calculate-salary-button');
+const $resultContent = document.querySelector('#result-values-salary');
+
+const $salaryFormTitle = document.createElement('h3');
+$salaryFormTitle.classList.add = "salary-form-title"
+$salaryFormTitle.innerText = "Enter the annual salary of each family member, only one salary per input";
+$salaryFormContent.insertBefore($salaryFormTitle, $salaryForm);
+
+$buttonAddInput.onclick = () => {
+    $salaryFormContent.style.display = "block";
+    $salaryFormTitle.style.display = "block";
+
+    const $salaryInputs = createLabelInput("Annual Salary: ", "salary-label", "salary-input");
+    $salaryForm.insertBefore($salaryInputs, $buttonCalculateSalary)
+
+    return false;
+}
+
+$buttonRemoveInput.onclick = () => {
+    const $allLabels = document.querySelectorAll('.salary-label');
+    const $allInputs = document.querySelectorAll('.salary-input');
+
+    $allLabels[$allLabels.length - 1].remove();
+    $allInputs[$allInputs.length - 1].remove();
+    $resultContent.style.display = "none";
+
+    cleanUpInput($allInputs);
+    return false;
+}
+
+$buttonCalculateSalary.onclick = () => {
+    const allWagesFromHtml = document.querySelectorAll('.salary-input');
```

Comment:
I think you should keep consistency with the names you use, meaning that you should change everything to ``wage`` or ``salary``

Reply:
Yes, it's better understood this way, 
also change the id of <p id="smaller-salary"> which was lower-salary, for the same reason.

I'm going to start implementing it (insert muscle emoji)